core/src/banking_stage.rs

```diff
@@ -734,7 +759,7 @@ impl BankingStage {
         let recorder = poh_recorder.lock().unwrap().recorder();
         let socket = UdpSocket::bind("0.0.0.0:0").unwrap();
         let mut buffered_packet_batches = VecDeque::with_capacity(batch_limit);
-        let banking_stage_stats = BankingStageStats::new(id);
+        let mut banking_stage_stats = BankingStageStats::new(id);
```

Comment:
still learning rust, a little confused on why we don't need this for atomicXYZ but we do for histogram

Reply:
Atomics have interior mutability, so they are not declared as `mut`.

Reply:
ok thanks for sharing! let me know if you'd prefer me to change it such that mut isn't required!

Reply:
You'll want to make sure that `cargo check` and `cargo clippy` both pass. You can see that the CI "checks" failed, and you can see the output [here](https://buildkite.com/solana-labs/solana/builds/63583#dae208d0-329c-4716-ac3d-8769a6bdc3d5). This log also includes the commands that are run, so you can mimic them locally to ensure this CI step passes.